### PR TITLE
Realize correct output of 'pem python-install help' command

### PR DESCRIPTION
**Pull request by ivan-podorozhnyi**
Issue fixed. 'pem python-install help' command works properly.
Affects #48

### CODE REVIEW

**Review by extsoft — CHANGES_REQUESTED** [review]
@ivan-podorozhnyi please rewrite commit message.

**Comment by ivan-podorozhnyi** [comment]
@extsoft done

File: tests/pem-python-install-help.bats [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bats
+
```
Comment (extsoft) [inline tests/pem-python-install-help.bats:2]: @ivan-podorozhnyi, you forget to connect commons.

**Review by extsoft — CHANGES_REQUESTED** [review]

**Review by extsoft — APPROVED** [review]

**Comment by extsoft** [comment]
@ivan-podorozhnyi merged.

**Pull request merged**